.werft/delete-preview-environment-cron.yaml

```diff
@@ -0,0 +1,54 @@
+pod:
+  serviceAccount: werft
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: dev/workload
+            operator: In
+            values:
+            - "builds"
+  volumes:
+  - name: gcp-sa
+    secret:
+      secretName: gcp-sa-gitpod-dev-deployer
+  - name: gcp-sa-release
+    secret:
+      secretName: gcp-sa-gitpod-release-deployer
+  containers:
+  - name: build
+    image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:prs-ws-man-bridge-audit.10
+    workingDir: /workspace
+    imagePullPolicy: Always
+    volumeMounts:
+    - name: gcp-sa
+      mountPath: /mnt/secrets/gcp-sa
+      readOnly: true
+    - name: gcp-sa-release
+      mountPath: /mnt/secrets/gcp-sa-release
+      readOnly: true
+    env:
+    - name: WERFT_HOST
+      value: "werft.werft.svc.cluster.local:7777"
+    - name: HONEYCOMB_DATASET
+      value: "werft"
+    - name: HONEYCOMB_API_KEY
+      valueFrom:
+        secretKeyRef:
+          name: honeycomb-api-key
+          key: apikey
+    command:
+      - bash
+      - -c
+      - |
+        sleep 1
+        set -Eeuo pipefail
+
+        sudo chown -R gitpod:gitpod /workspace
+
+        (cd .werft && yarn install && mv node_modules ..) | werft log slice prep
+        printf '{{ toJson . }}' > context.json
+
+        # We're merging with the script commented so we can safely test prior to fully commit to it
+        npx ts-node .werft/delete-preview-environments-cron.ts
```

Comment:
There's a typo here I'm afraid ☺️ The file name is .werft/delete-preview-environment-cron.ts (but I agree that it would be nicer if the file was named `delete-preview-environments-cron.ts` so maybe just rename the file ☺️

Reply:
Oh my gosh this is getting more frequent than I'd like to admit 🤦